samples/todolist.md

```diff
@@ -0,0 +1,160 @@
+1. Create the pelorus namespace
+```
+oc create namespace pelorus
+```
+2. Install the required granfa and prometheus operators
+```
+helm install operators charts/operators --namespace pelorus
+```
+3. Create a GitHub Personal Access Token and store it as a secret in Openshift:
+```
+oc create secret generic github-secret --from-literal=GIT_USER=<username> --from-literal=GIT_TOKEN=<personal access token> --namespace pelorus
+```
+4. Install Pelorus
+```
+helm install pelorus charts/pelorus --namespace pelorus
+```
+5. Check the install
+```
+oc get all -n pelorus
+```
+***Note*** pause to allow the install to fully complete.
+
+## Pelorus configuration
+1. Make a copy of the values.yaml file [provided in this repo](https://github.com/konveyor/pelorus/blob/master/charts/pelorus/values.yaml) and save it to /var/tmp/values.yaml
+```
+cp charts/pelorus/values.yaml /var/tmp/
+```
+2. Update the /var/tmp/values.yaml file with the following example:
+```
+exporters:
+  instances:
+  - app_name: deploytime-exporter
+    source_context_dir: exporters/
+    extraEnv:
+    - name: APP_FILE
+      value: deploytime/app.py
+    - name: LOG_LEVEL
+      value: DEBUG
+    - name: NAMESPACES
+      value: mongo-persistent
+    source_ref: master
+    source_url: https://github.com/your_org/pelorus.git
+  - app_name: committime-generic
+    env_from_secrets:
+    - github-secret
+    source_context_dir: exporters/
+    extraEnv:
+    - name: APP_FILE
+      value: committime/app.py
+    - name: LOG_LEVEL
+      value: DEBUG
+    source_ref: master
+    source_url: https://github.com/your_org/pelorus.git
+```
+3. Documentation regarding values.yaml can be found [here](https://github.com/konveyor/pelorus/blob/master/docs/Configuration.md).
+  Apply the updated values for Pelorus by executing:
+```
+helm upgrade pelorus charts/pelorus --namespace pelorus --values /var/tmp/values.yaml
+```
+***Note*** pause to allow the committime exporter pod to build and deploy.
+
+Pelorus should now be installed and configured to measure the todolist sample app. We'll have to deploy the sample application to view measurements from Pelorus's Grafana dashboard.
+
+## Todolist sample application install steps
+
+1. git clone the forked copy of the
+```
+git clone https://github.com/your_org/mig-demo-apps.git
+```
+2. Install the todolist-mongo-go sample application
+```
+cd mig-demo-apps/apps/todolist-mongo-go
+export GITHUB_ORG=<YOUR_REAL_GITHUB_FORK_ORG>
+sed -i 's/your_org/${GITHUB_ORG}/g' mongo-persistent.yaml
+oc create -f mongo-persistent.yaml
+```
+
+The todolist application and mongo database should now build and deploy into the mongo-persistent namespace.
+
+3. Check the build
+```
+oc get all -n mongo-persistent
+```
+***Note*** pause to allow the todolist pod to build and deploy.
+
+## View the Pelorus measurements
+
+1. In your OpenShift Pelorus project page, open the link to granfa or get the link via the cli:
+```
+oc get route -n pelorus | grep grafana
```

Comment:
`oc get route grafana-route -o=go-template='https://{{.spec.host | printf "%s\n" }}'` is cleaner